spec/menu_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'menu'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.